src/gsi/read_fl_hdob.f90

```diff
@@ -1121,7 +1121,7 @@ subroutine read_fl_hdob(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,si
               cdata_all( 1,iout)=woe                    ! wind error
               cdata_all( 2,iout)=dlon                   ! grid relative longitude             
               cdata_all( 3,iout)=dlat                   ! grid relative latitude                  
-              cdata_all( 4,iout)=dlnpsob                ! ln(surface pressure in cb)
+              cdata_all( 4,iout)=missing                ! ln(surface pressure in cb) !Since dlnpsob is not provided by SFMR, force it to be missing. Not used in setupspd.f90
```

Comment:
where is "missing" defined?  Could we directly use r_missing defined in the constants module?

Reply:
Sure @TingLei-NOAA , I've just modified the missing to r_missing from the constants module.

Reply:
@XuLu-NOAA  Thanks.